OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs

```diff
@@ -44,7 +44,7 @@ public void ReturnToBase(Actor airfield = null)
 			if (isPlane)
 				Self.QueueActivity(new ReturnToBase(Self, false, airfield));
 			else
-				Self.QueueActivity(new HeliReturnToBase(Self, false));
+				Self.QueueActivity(new HeliReturnToBase(Self, false, airfield));
```

Comment:
Minor: Is it safe to fix the `Desc` tag and parameter to not use `airfield` but something generic like `destination`?